### ISSUE TITLE
sudo apt-get fails 

### ISSUE THREAD

**Issue by Sindorman** [travis-ci/travis-ci#9851]
Hello, my team has recently got stuck on this problem. Our builds fail with the following message:
```
 sudo -E apt-get -yq --no-install-suggests --no-install-recommends --allow-downgrades --allow-remove-essential --allow-change-held-packages install robosub
E: Command line option --allow-downgrades is not understood
The command "sudo -E apt-get -yq --no-install-suggests --no-install-recommends --allow-downgrades --allow-remove-essential --allow-change-held-packages install robosub" failed and exited with 100 during .
```

Here is our config:
```
{
  "os": "linux",
  "env": "BUILD=true",
  "dist": "trusty",
  "sudo": "required",
  "group": "deprecated-2017Q2",
  "addons": {
    "apt": {
      "sources": [
        {
          "key_url": "https://raw.githubusercontent.com/ros/rosdistro/master/ros.key ",
          "sourceline": "deb http://packages.ros.org/ros/ubuntu trusty main"
        },
        {
          "key_url": "http://robosub.eecs.wsu.edu/repo/repository_key",
          "sourceline": "deb http://robosub.eecs.wsu.edu/repo/ /"
        },
        {
          "key_url": "http://packages.osrfoundation.org/gazebo.key",
          "sourceline": "deb http://packages.osrfoundation.org/gazebo/ubuntu trusty main"
        }
      ],
      "packages": [
        "robosub"
      ]
    }
  },
  "script": [
    "if [ \"$BUILD\" == \"true\" ] ; then catkin_make; fi",
    "if [ \"$TEST\" == \"true\" ] ; then catkin_make; source devel/setup.bash; catkin_make run_tests; catkin_test_results; fi",
    "if [ \"$LINT\" == \"true\" ] ; then catkin_make roslint; fi"
  ],
  "install": [
    "if [ \"$TEST\" == \"true\" ] ; then sudo apt-get install robosub-simulator; fi"
  ],
  "language": "generic",
  "before_script": [
    "source /opt/ros/indigo/setup.bash",
    "cd ../ && mkdir src && mv robosub src/ && cd src",
    "git clone https://github.com/PalouseRobosub/robosub_msgs.git",
    "catkin_init_workspace",
    "cd ..",
    "if [ \"$TEST\" == \"true\" ] ; then cd src/; git clone https://github.com/PalouseRobosub/robosub_simulator.git; cd ../; fi"
  ]
}
```

**Comment by chrisvanwormer** [comment]
As of Friday, July 6th, our team has encountered the same problem.  Based on some research, it appears that trusty installs apt-get version 1.0.1 by default and the --allow-downgrades option is not supported with that version.  

This morning while troubleshooting some more we recently restarted a build that passed last week before we encountered this issue and that same build is failing this morning. This makes us think something changed on travis and it's not related to any changes we've made in the last week.

We're trying to find some way to get our builds moving again.
